Controller/MediaAdminController.php

```diff
@@ -111,4 +112,26 @@ public function uploadAction()
         ));
     }
 
+    /**
+     * Sets the admin form theme to form view. Used for compatibility between Symfony versions.
+     *
+     * @param FormView $formView
+     * @param string   $theme
+     */
+    protected function setFormTheme(FormView $formView, $theme)
+    {
+        /** @var \Twig_Environment $twig */
+        $twig = $this->get('twig');
+        if ($twig->hasExtension('Symfony\Bridge\Twig\Extension\FormExtension')){
```

Comment:
You can use a guard clause (add a `return` statement at the end of this block and remove the `else`).